src/Repository/DefaultUnleashRepository.php

```diff
@@ -538,4 +493,71 @@ private function parseVariants(array $variantsRaw): array
 
         return $variants;
     }
+
+    /**
+     * @throws ClientExceptionInterface
+     * @throws InvalidArgumentException
+     *
+     * @return void
```

Comment:
There are two returns, delete this one, please.

Reply:
Fixed